BlogTemplate/Pages/Post.cshtml.cs

```diff
@@ -29,6 +29,10 @@ public void OnGet([FromRoute] int id)
             {
                 RedirectToPage("/Index");
             }
+            if(Post.Comments.Count > 99)

```

Comment:
Don't use 99 as a magic number.  Put it in a constant, e.g. `const int MaxAllowedComments = 100;`

That also makes it easier to refactor later if we want to, for example, put this into a Site Settings type page.